packages/editor/src/components/post-title/index.js

```diff
@@ -129,12 +132,13 @@ const applyWithSelect = withSelect( ( select ) => {
 	const { getEditedPostAttribute, getEditorSettings } = select( 'core/editor' );
 	const { getPostType } = select( 'core' );
 	const postType = getPostType( getEditedPostAttribute( 'type' ) );
-	const { titlePlaceholder } = getEditorSettings();
+	const { titlePlaceholder, hasFixedToolbar } = getEditorSettings();
```

Comment:
If removing the fixed toolbar option, should we rename hasFixedToolbar property to isFocusModeEnable?

Reply:
@afercia mentioned that the mode should probably not be called "Focus Mode" because it could cause confusion with accessibility tools that have modes with the same name, so I would hold off on renaming variables until we know what the mode should be called.